src/resources/LoadLocalizations/LoadLocalizations.tsx

```diff
@@ -9,7 +9,18 @@ interface Props {
 }
 
 const LoadLocalizations = ({ children }: Props) => {
-  const backupLang = getCookie('i18next') ? getCookie('i18next') : 'no_nb';
+  const getBackupLanguage = () => {
+    const value = getCookie('i18next');
+    if (value === 'no_nn') {
```

Comment:
Just trying to understand this code block. Are you trying to return a default language? Doesn't BFF already defaults language to bokmål?

Reply:
It's meant as a second verification for i18next cookie corresponding with actual localization file names

Reply:
But you are returning the same text as the text in "if" condition.

Reply:
Yeah that was redundant, changed it to just return value now